alerter/src/main/java/com/tapadoo/alerter/Alert.java

```diff
@@ -53,6 +53,7 @@
     private long duration = DISPLAY_TIME_IN_SECONDS;
 
     private boolean enableIconPulse = true;
+    private boolean enableInfiniteDuration = false;
```

Comment:
In Java, booleans have a default value of `false` - so you can remove the `= false;`.